### DISCUSSION TITLE
Minimum configuration required to configure OpenMetadata for a POC

### DISCUSSION THREAD

**Discussion by ShilpaVernekar** [open-metadata/OpenMetadata#5413] [Q&A] [answered]
What is the minimum configuration necessary to configure OpenMetadata for a POC? 
Would it be:
- Ubuntu 22.04
- t2.medium (Amazon EC2)

**Comment by ShilpaVernekar** [comment, marked as answer]
For a Docker based installation with all the components running locally, the following would be required:
- OpenMetadata (latest version)
- MySQL or Postgres as the database
- Elasticsearch
- Airflow
- Also require around 6 GB RAM
So it is recommended to use t2.large (Amazon EC2)

Please [refer the docs](https://docs.open-metadata.org/deploy/local-deployment#requirements-osx-and-linux) for further details.

Even if you don't need the search ability for your use case, **Elasticsearch** is a required component. OpenMetadata relies on Elasticsearch for a variety of functions. For example, when assigning an owner to a table, or assigning tags, we use Elasticsearch to generate the auto-suggestions in the dropdown.

**Reply by OneCyrus** [reply]
is an airflow instance required for a PoC or can I disable it and still use open metadata without UI generated ingestion workflows?
